src/HelpText.cpp

```diff
@@ -332,7 +332,7 @@ const wxString VerCheckHtml()
    wxTextOutputStream s(o);
    s
       << "<center>[["
-      << "https://github.com/saucedacity/saucedacity/releases|"
+      << "https://github.com/tenacity/tenacity/releases|"
```

Comment:
CAREFUL!!!

Reply:
Wait, this is also wrong because of the username and not just because of the reference to a web resource.

Reply:
Also an automation error. Will definitely be fixing this!